### PR TITLE
Fix PVS-Studio warnings, part 2

### PR DESCRIPTION
**Pull request by cher-nov**
Separated from #54. See commit descriptions for details.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -123,14 +123,13 @@ bool command_handler_c::parse(int argc, char *argv[]) {
...
-        std::cout << spawner->help();
-        return true;
-    }
-    if (spawner && spawner->init()) {
-        spawner->run();
-    } else {
-        std::cout << spawner->help();
+    if (spawner) {
+        if (!show_help) { show_help = !spawner->init(); }
+        if (show_help) {
```
Comment (klenin) [inline src/main.cpp]: Is there a reason not to use `&&` ?

File: libspawner/inc/buffer.h [hunk 2]
```diff
@@ -116,12 +116,7 @@ class output_stdout_buffer_c
 
 void dprintf(const char* format, ...);
 
-class input_stdin_buffer_c
-    : public input_buffer_c
-    , protected handle_buffer_c {
```
Comment (klenin) [inline libspawner/inc/buffer.h]: These are unrelated formatting changes, please remove

File: libspawner/src/win32/pipes.cpp [hunk 3]
```diff
@@ -1,233 +1,228 @@
-#include "pipes.h"
```
Comment (klenin) [inline libspawner/src/win32/pipes.cpp]: Please remove 200 lines of diff below

File: src/main.cpp (hunk 1 above)
Comment (cher-nov) [inline src/main.cpp]: See above. `spawner` was once utilized before actual check.

File: libspawner/src/win32/pipes.cpp (hunk 3 above)
Comment (cher-nov) [inline libspawner/src/win32/pipes.cpp]: Goddamned CRLF->LF conversion. Sorry.
